frontend/src/DocumentUploadForm.js

```diff
@@ -33,16 +34,31 @@ class DocumentUploadForm extends React.Component {
       } else {
         this.setState({ goToChapterSelect: true, data: res.data });
       }
+    }).catch(error => {
+      console.log(error)
+      this.setState({
+          error: true,
+          uploading: false    
+      }) 
     });
   }
 
   componentDidMount() {
-    brevityHttpGet('/auth/user').then(res => this.setState({ userName: res.data }));
+    let userName = localStorage.getItem('username');
+    if (userName === null) {
+      brevityHttpGet('/auth/user').then(res => localStorage.setItem('username', res.data));
+    }
   }
 
   render() {
     return (
       <div>
+        { this.state.error && 
+          <Alert variant="danger" onClose={() => { this.setState({ error: false })}} dismissible>
+            <Alert.Heading>Error</Alert.Heading>
+            <p>You have uploaded a file that is larger than 80MB, please try uploading something under that</p>
+          </Alert>
+        }
```

Comment:
Could we move this `Alert` into the `Col`
Change the `<p>` to `<span>`
Change the heading string to "Upload failed"
Shorten the message string to "File is larger than 80 MB. Please try again with a smaller file."
Add this style for the alert heading
```
.alert-heading {
    font-size: 16px;
    font-weight: bold;
}
```

Should look like this in the end:
![Screenshot from 2020-03-09 10-40-37](https://user-images.githubusercontent.com/15786604/76225458-e75f8c00-61f2-11ea-8e78-cf78b7bc3269.png)